amplify/backend/api/YourClearWayAPI/schema.graphql

```diff
@@ -89,6 +82,23 @@ enum Tag {
     WATER
 }
 
+enum Difficulty {
+    BEGINNER
+    INTERMEDIATE
+    ADVANCED
+}
+
+enum Cost {
+    ONE
+    TWI
+    THREE
+    FOUR
+}
+
+enum Type {
+    HOME_IMPROVEMENT
+    GREEN_LIFESTYLE
+}
```

Comment:
nit: on the mobile app measure list, there isn't a `GREEN_LIFESTYLE` type, but there is a `BEHAVIORAL` type. Not sure if we want to include both?

also maybe rename `Type` to something more specific for its use case since "type" can mean many things (maybe `QuestionType`)?

Reply:
the keyword in appsync is _typename but QuestionType is definite better